lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -736,10 +734,7 @@
     /* ***
      * Ideally we want to handle functionality with JS.
      * This functionality should eventually be moved into CS/JS, and out of here. */
-    .has-hint:hover > .hint {
-      @include left($baseline*10);
-    }
-
+    .has-hint:hover > .hint,
     .has-hint input:focus ~ .hint {
```

Comment:
We need display block here, right?

Reply:
@Lunyachek can you address this comment?